### PR TITLE
feat: updated order details totals section styling

### PR DESCRIPTION
**Pull request by AlexanderDevitsky**
## Related Issue

https://vsf.atlassian.net/browse/M2-508

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/api-client/src/api/customerOrders/customerOrders.ts [hunk 1]
```diff
@@ -6,8 +6,10 @@ export default gql`
       orders(currentPage: $currentPage, filter: $filter, pageSize: $pageSize) {
         items {
           order_number
+          number
           id
           created_at
```
Comment (bartoszherba) [inline packages/api-client/src/api/customerOrders/customerOrders.ts:11]: You can remove `created_at`

**Review by bartoszherba — CHANGES_REQUESTED** [review]
Build fails at your branch after css changes. Something is wrong with parsing new styles. Please, verify that build pass `yarn build`. You can check issue details in PR build checks.

+ One small comment to remove field from the request.

File: packages/api-client/src/api/customerOrders/customerOrders.ts (hunk 1 above)
Comment (AlexanderDevitsky) [reply packages/api-client/src/api/customerOrders/customerOrders.ts]: @bartoszherba I caught the style error and fixed it. I also noticed that the new SingleOrder.vue component was created and already made all the new changes to this component.

File: packages/theme/modules/customer/pages/MyAccount/OrderHistory/OrderHistory.vue [hunk 2]
```diff
@@ -43,7 +43,7 @@
             v-for="order in orders"
             :key="order.order_number"
           >
-            <SfTableData>{{ order.order_number }}</SfTableData>
+            <SfTableData>{{ orderGetters.getId(order) }}</SfTableData>
```
Comment (bartoszherba) [inline packages/theme/modules/customer/pages/MyAccount/OrderHistory/OrderHistory.vue:46]: Please, revert that. We are trying to get rid of getters whenever it is possible to reduce the bundle size, especially if getter does not introduce any intermediate logic but is a pure object prop accessor like in this case.

File: packages/theme/modules/customer/pages/MyAccount/OrderHistory/OrderHistory.vue [hunk 3]
```diff
@@ -52,7 +52,7 @@
             <SfTableData class="orders__view orders__element--right">
               <SfLink
                 data-cy="order-history-btn_view"
-                :link="localeRoute({ name: 'customer-single-order', params: { orderId: order.order_number } })"
+                :link="localeRoute({ name: 'customer-single-order', params: { orderId: orderGetters.getId(order) } })"
```
Comment (bartoszherba) [inline packages/theme/modules/customer/pages/MyAccount/OrderHistory/OrderHistory.vue:55]: Same here, as in comment above.

**Review by bartoszherba — CHANGES_REQUESTED** [review]
Last changes.

File: packages/theme/modules/customer/pages/MyAccount/OrderHistory/OrderHistory.vue (hunk 3 above)
Comment (AlexanderDevitsky) [reply packages/theme/modules/customer/pages/MyAccount/OrderHistory/OrderHistory.vue]: @bartoszherba Thank you for the info. Reverted these changes to the previous state

**Review by bartoszherba — APPROVED** [review]

**Pull request merged**